bool highlight_node
int32 node_index
---
bool success
string msg
